aikau/src/main/resources/alfresco/core/Page.js

```diff
@@ -319,17 +320,16 @@ define(["alfresco/core/ProcessWidgets",
 
          if (this.publishOnReady)
          {
-            array.forEach(this.publishOnReady, lang.hitch(this, "onReadyPublish"));
+            array.forEach(this.publishOnReady, lang.hitch(this, this.onReadyPublish));
          }
 
          // Release all publications that have been queued awaiting all the widgets to have finished being
          // created...
          PubQueue.getSingleton().release();
-         this.alfPublish("ALF_WIDGETS_READY", {});
+         this.alfPublish(topics.PAGE_WIDGETS_READY, {});
```

Comment:
I know it's existing code, but you don't need to send through an empty object (it'll create one automatically). Maybe for future reference, rather than changing this branch again?